app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -597,9 +597,15 @@ public void onResume() {
 
         // Display full text on click in case it doesn't fit in a single line
         binding.cardIdView.setOnClickListener(v -> {
+            TextView cardIdView = new TextView(LoyaltyCardViewActivity.this);
+            cardIdView.setText(loyaltyCard.cardId);
+            cardIdView.setTextIsSelectable(true);
+            int contentPadding = getResources().getDimensionPixelSize(R.dimen.alert_dialog_content_padding);
+            cardIdView.setPadding(contentPadding, contentPadding, contentPadding, 0);
```

Comment:
This padding is inconsistent with padding of other dialogs:

https://github.com/CatimaLoyalty/Android/blob/16b3fbdf9c4289e00bab95a76797c4fc7487cd3d/app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java#L347-L352

```suggestion
            cardIdView.setPadding(contentPadding, 0, contentPadding, 0);
```